eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -661,4 +662,13 @@ private function prependTranslatorConfiguration(ContainerBuilder $container)
             $container->prependExtensionConfig('framework', ['translator' => ['paths' => [$translationsPath]]]);
         }
     }
+
+    private function registerUrlWildcardsConfiguration(array $config, ContainerBuilder $container)
+    {
+        if (!isset($config['url_wildcards']['enabled'])) {
+            $config['url_wildcards']['enabled'] = false;
+        }
+
+        $container->setParameter('ezpublish.url_wildcards.enabled', $config['url_wildcards']['enabled']);
```

Comment:
```suggestion
        $container->setParameter('ezpublish.url_wildcards.enabled', $config['url_wildcards']['enabled'] ?? false);
```
and you can drop lines 668-671